x-pack/plugins/maps/public/kibana_services.d.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IIndexPattern } from 'src/plugins/data/public';
+
+export function getLicenseId(): any;
+export function getInspector(): any;
+export function getFileUpload(): any;
+export function getIndexPatternSelect(): any;
```

Comment:
`getFileUpload()` and `getIndexPatternSelect()` do not seem to be used (?). If so, I'd just remove here and the implementation in `kibana_sercvices.js`